src/wasm-builder.h

```diff
@@ -445,6 +445,15 @@ class Builder {
     return makeConst(value);
   }
 
+  // Module-level helpers
+  static Global* makeGlobal(Name name, WasmType type, Expression* init, bool mutable_) {
```

Comment:
you could make the last argument an enum, and then you wouldn't need the comments everywhere.

Reply:
Good idea, my instincts here were wrong.

`NotMutable` or `Immutable`?

Reply:
I'd say `Immutable`

Reply:
Ok, also a search shows that's more common, updated.